src/views/FileEditView.vue

```diff
@@ -165,19 +152,39 @@ const getFileSchema = () => {
 
   <div
     v-if="fileContent !== null"
-    :class="`bg-white fill-height px-12 py-8 d-block file-editor ${!isFormJSON && 'file-editor-code'}`"
+    :class="`bg-white px-12 py-8 d-block file-editor ${!isFormJSON && 'file-editor-code'}`"
   >
     <h2>{{ session.title }}</h2>
     <p>{{ filePath }}</p>
 
-    <div>
-      <eox-jsonform
-        :schema="isFormJSON ? JSON.parse(fileContent) : getFileSchema()"
-        :noShadow="false"
-        :unstyled="false"
-        @change="onFileChange"
-      ></eox-jsonform>
-    </div>
+    <v-row no-gutters :class="previewURL ? 'd-flex mt-6' : ''">
+      <v-col v-if="previewURL" class="mt-2 file-preview">
```

Comment:
Can we have the **preview on the right hand side** instead of left? I have the impression the user would understand better if the editor is always left (with or without preview).

Also, maybe we can **remove**  `<h2>{{ session.title }}</h2>` and `<p>{{ filePath }}</p>` to make more room. Now that we have preview on the side, we can also **get rid of the "preview" button** on top. As for the "preview" button in the file list, we could disable it for now so users are not confused (it currently doesn't do anything).
Mockup:
![image](https://github.com/user-attachments/assets/1de5e787-830d-4ea4-a786-86a4e7a0491d)